examples/lstm_stateful.py

```diff
@@ -0,0 +1,204 @@
+'''Example script showing how to use stateful RNNs
+to model long sequences efficiently.
+Depending on the parameters, either only the stateful LSTM converges,
+or both the stateful and stateless LSTM converge.
+'''
+from __future__ import print_function
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+# -------------------
+# EDITABLE PARAMETERS
+# -------------------
+
+# timesteps to use in the output averaging of the input
+# e.g. use 2 for 2-point average
```

Comment:
Please add more info, it is not possible to understand this comment without more context.